data/sql_updates/create_candidate_detail_view.sql

```diff
@@ -54,6 +54,13 @@ from dimcand
         select distinct on (cand_sk) * from dimcandproperties
             order by cand_sk, candproperties_sk desc
     ) cand_p_most_recent using(cand_sk, cand_id)
+    -- there are some holes in this data so we want the lat time it was updated
+    left join(
+        select distinct on (cand_sk, cand_ici_desc, cand_ici_desc)
+            cand_sk, cand_ici_desc, cand_ici_desc
+        where cand_ici_desc != Null
+        order by  cand_sk, cand_id, election_yr desc
+    ) full_ici using (cand_sk, cand_id)
```

Comment:
Just curious, but why `...using (cand_sk, cand_id)`? Is `...using (cand_sk)` not enough?